packages/edit-post/src/components/editor-regions/style.scss

```diff
@@ -1,47 +1,74 @@
+// We have to hide admin-bar in editor on mobile, in order to make the Mobile Safari experience bearable.
+// These two rules need to be moved to somewhere appropriate, and the !importants removed.
+#wpadminbar {
+	display: none;
+	@include break-small() {
+		display: initial;
+	}
+}
+#wpbody {
+	padding-top: 0 !important;
+	@include break-small() {
+		padding-top: initial !important;
+	}
+}
+
 .edit-post-editor-regions {
 	display: flex;
 	flex-direction: column;
 	height: 100%;
 	max-height: 100%;
-	position: relative;
 
-	// On Mobile keep a margin for the header and admin header
-	// as both of these are fixed
+	// Fill the entire screen on Mobile.
+	// We don't show the WP-adminbar, as editing on Mobile Safari is barely usable with it present.
+	position: fixed;
 	top: 0;
-	@include break-medium() {
-		margin-top: 0;
+	left: 0;
+	right: 0;
+	bottom: 0;
 
-		// On Desktop position the container as fixed to avoid scroll bleed.
-		position: fixed;
-		top: $admin-bar-height;
-		left: 0;
-		right: 0;
-		bottom: 0;
+	// Cancel the fixed positioning used on Mobile.
```

Comment:
The comment is a bit misleading as we're not canceling the fixed positioning but more adjustinig the "top" to show the WPAdmin header right?